rpcs3/Emu/Cell/lv2/sys_process.cpp

```diff
@@ -402,6 +402,22 @@ void _sys_process_exit2(ppu_thread& ppu, s32 status, vm::ptr<sys_exit2_param> ar
 
 	// TODO: set prio, flags
 
+	_sys_process_restart(ppu, argv, envp, data);
+}
+
+void sys_process_exit3(ppu_thread& ppu, s32 status)
+{
+	ppu.state += cpu_flag::wait;
+
+	sys_process.warning("_sys_process_exit3(status=%d)", status);
+
+	return _sys_process_exit(ppu, status, 0, 0);
+}
+
+void _sys_process_restart(ppu_thread& ppu, std::vector<std::string>& argv, std::vector<std::string>& envp, std::vector<u8>& data)
```

Comment:
maybe call it lv2_exitspawn, and use std::span instead of vectors.

Reply:
Done.